specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/publicIpAddress.json

```diff
@@ -358,6 +358,10 @@
           "$ref": "#/definitions/PublicIPAddressDnsSettings",
           "description": "The FQDN of the DNS record associated with the public IP address."
         },
+        "ddosSettings": {
+          "$ref": "#/definitions/DdoSSettings",
```

Comment:
This should probably be ./networkInterface.json#/definitions/DdoSSettings, we should validate